.github/workflows/deutschland_generator.yaml

```diff
@@ -31,4 +32,14 @@ jobs:
           upload-to-testpypi: false
           pypi-token: ${{ secrets.PYPI_PRODUCTION }}
           testpypi-token: ${{ secrets.PYPI_TEST }}
-          python-version: ${{ matrix.python-version }}
\ No newline at end of file
+          python-version: ${{ matrix.python-version }}
+
+      - name: "Generate deutschland code"
+        if: github.repository_owner !== 'bundesAPI'
```

Comment:
Can we shorten this to:
```
commit-to-git:  ${{ github.repository_owner == 'bundesAPI'}}
upload-to-pypi: ${{ github.repository_owner == 'bundesAPI'}}
upload-to-testpypi: false
```
within one block?

Reply:
Yes, looks like in one block is working.  I would prefer this instead an disabling the workflow completely. 

Sidenote: I had the syntax wrong on the initial commits -> `!==` needs to be `!=` in Actions. But since we now only use the short version now, we do not need this anymore.